backend/src/controllers/auth-controller.ts

```diff
@@ -51,8 +79,21 @@ export const loginUser = async (req: Request, res: Response) => {
         const password = req.body.password as string;
         User.findOne({email: email})
```

Comment:
Future note: In `async` JavaScript, avoid using `.then()` and `.catch()` syntax. Use `try {} catch {}` and `async await` instead.

Reply:
try {
    const user = await User.findOne({email: email})
} catch (error) {
    res.status(500).json({message: ""})
}